src/CopyrighFixer/CopyrighFixer/owner.cpp

```diff
@@ -20,12 +20,38 @@ const QString &Owner::getOwnerName() const {
     return _name;
 }
 
-void Owner::setTimeRange(const QString &interval) {
-    _timeRange = interval;
+void Owner::setTimePoint(const int &interval) {
```

Comment:
The size of the int type is 4 byte but size of the const reference to int is 8 byte.
please replace const int& to int
```suggestion
void Owner::setTimePoint(int interval) {
```

Please do not use the reference for the atomic types.